src/transport/HttpClient.ts

```diff
@@ -278,7 +278,7 @@ class RealHttpClient implements HttpClientBase {
     }
 
     putDocumentNotePageUpdate(workspaceId: string, noteId: string, content: string): Promise<void> {
-        return this.put(`/notes/${noteId}/document/pages/1`, {content: JSON.stringify(content)});
+        return this.put(`/notes/${noteId}/document/pages/1`, {content: content});
```

Comment:
i guess that there always should be 1 for now?

Reply:
Until we implement option to have multiple pages on UI there is no reason to pass the other number in this call